main/acle.md

```diff
@@ -1104,9 +1108,10 @@ This macro is defined for AArch32 only.
 ### 32-bit SIMD instructions
 
 `__ARM_FEATURE_SIMD32` is defined to 1 if the 32-bit SIMD instructions
-are supported and the intrinsics defined in [ssec-32SIMD](#simd-intrinsics-for-32-bit-operations) are
-available. This also implies support for the GE global flags which
-indicate byte-by-byte comparison results.
+are supported and the intrinsics defined in
+[ssec-32SIMD](#32-bit-simd-operations) are available. This also
```

Comment:
Wouldn't this section link not work when  the document is converted to pdf?

Reply:
Ah - good catch - the hyperlink was taking to the entry of the section in the TOC, not in the body of the specs. I have moved the `span` right before the sectioning, now it works. You can check in the artifacts of the latest build.